go.mod

```diff
@@ -1,8 +1,6 @@
 module github.com/stripe/stripe-cli
 
-go 1.22.0
-
-toolchain go1.23.5
```

Comment:
I think we don't need this anymore since we'll use the latest version so they both match